week2/homework-week2/Program.cs

```diff
@@ -0,0 +1,97 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+
+// 1. Calculator
+app.MapGet("/calculate", (string number1, string number2, string operation) => {
+    var number1Parse = int.TryParse(number1, out int value1);
+    var number2Parse = int.TryParse(number2, out int value2);
+  
+    if(!number1Parse || !number2Parse){
+        return Results.Ok(new { Error = $"Unable to parse input '{number1}', '{number2}'!" });  

```

Comment:
Maybe you should use the BadRequest instead of Ok to inform the user that there is an error

Reply:
yes I will do that